docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+
+```
+{ .http.status >= 400 && .http.status < 500 }
+```
+
+Find all traces where the `http.method` attribute is either `GET` or `DELETE`:
+
+```
+{ .http.method =~ “DELETE|GET” }
+```
+
+### Field expressions
+
+Fields can also be combined in various ways to allow more flexible search criteria. A field expression is a composite of multiple fields that define all of the criteria that must be matched to return results.
+
+#### Examples
+
+Find traces with "success" http status codes:
+
+```
+{ .http.status >= 200 && .http.status < 300 }
+```
+
+Find traces where a `DELETE` HTTP method was used and the instrinsic span status was not OK:
+
+```
+{ .http.method = "DELETE" && status != ok }
+```
+
+Both expressions require all conditions to be true on the same span. The entire expression inside of a pair of `{}` must be evaluated as true on a single span for it to be included in the result set.
+
+In the above example, if a span includes an `.http.method` attribute set to `DELETE` where the span also includes a `status` attribute set to `ok`, the trace would not be included in the returned results.
+
+## Combining spansets
+
+Spanset operators let you combine two sets of spans using and (`&&`) as well as union (`||`).
+
+- `{condA} && {condB}`
+- `{condA} || {condB}`
+
+
+For example, if you want to find a trace that went through two specific regions:
+
+```
+{ .region = "eu-west-0" } && { .region = "eu-west-1" }
+```
+
+Note the difference between the above and the following:
+
+```
+{ .region = "eu-west-0" && .region = "eu-west-1" }
+```
+
+The second expression returns no traces because it's impossible for a single span to have a `.region` attribute that is set to both region values at the same time.
+
+## Aggregators
+
+So far, all of the example queries expressions have been about individual spans. You can use aggregate functions to ask questions about a set of spans. These currently consist of:
+
+- `count` - The total count across a given intrinsic or items.
+- `avg` - The average across a given intrinsic or items.
+
+Aggregate functions allow you to carry out operations on matching results to further refine the traces returned. For more information on planned future work, refer to [How TraceQL works]({{< relref "architecture" >}}).
+
+For example, to find traces where the total number of spans is greater than `10`:
+
+```
+count() > 10
+```
+
+Find traces where the average duration of the spans in a trace is greater than `20ms`:
+
+```
+avg(duration) > 20ms
+```
+
+For example, find traces that have more than 3 spans with an attribute `http.status` with a value of `200`:
+
+```
+{ .http.status = 200 } | count() > 3
+```
+
+## Examples
+
+Find any trace with a span attribute or resource attribute `namespace` set to `prod`:
+
+```
+{ .namespace = "prod" }
+```
+
+Find any trace with a span attribute `namespace` set to `prod`:
```

Comment:
change lines 223 and 226 to be have the resource scope instead of the span scope. it makes more sense.

Reply:
Done.